client_tls.go

```diff
@@ -103,7 +114,7 @@ func (c *Client) haveTLSClientConfig() *tls.Config {
 // If path is a directory then scans for files recursively. If path is not set then defaults to /etc.
 // File name should match *.crt or *.pem.
 func (c *Client) TLSRootCerts(path string) *Client {
-	c.haveTLSClientConfig().RootCAs = NewCertPool(path)
+	c.haveTLSClientConfig().RootCAs = NewCertPool(path, false)
```

Comment:
Instead of wired-in `false`, it may make sense to have parameter `loadSystemCerts`. That is not backward compatible. But lets the client developer choose if system CAs are to be supported.

Note that the provided path may contain several PEM files, including of public CAs. But a bit more consistent with server's `TLSClientCert`.

Reply:
Probably name `addSystemCerts` would be more descriptive.